src/core.jsx

```diff
@@ -60,6 +61,7 @@ export default (
 
       state = {
         props: {},
+        pastDelay: false,
```

Comment:
- use `print` field instead of `pastDelay`
 - init it to `true`
 - in the `render` method you can simplify the condition : `if (!this.state.print) return null`